css/jquery.terminal.css

```
@@ -1185,6 +1209,25 @@ terminal .terminal-output > div {
+ */
+.terminal-debug .cmd-clipboard, .terminal-debug .cmd-editable {
+    color: var(--color, #ccc) !important;
+    outline: 1px solid #ccc !important;
+    font-size: var(--terminal-font-size) !important;
+}
+.terminal-debug .cmd-clipboard {
+    position: static !important;
+    text-indent: 0 !important;
+    width: 100% !important;
+    height: calc(var(--terminal-font-size) * 1px) !important;
+}
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)